pkg/globalnet/controllers/node_controller.go

```diff
@@ -52,21 +52,34 @@ func NewNodeController(config *syncer.ResourceSyncerConfig, pool *ipam.IPPool, n
 		nodeName:                   nodeName,
 	}
 
+	clusterCIDR := ""
+	if len(clusterCIDRs) > 0 {
```

Comment:
Don't we want to report an error if  len(clusterCIDRs) isn't bigger than 0?

Reply:
yeah we could although the operator would never pass empty cluster CIDRs anyway. I have a follow-up PR to check all `clusterCIDRs` so this check will go away anyway.